### PR TITLE
Create ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60261](https://cumulocity.atlassian.net/browse/MTM-60261)
Original PR: [6838](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6838)

### CODE REVIEW

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Changes to unit in the dp template is now reflected in the info gauge widget view. (#6823) [GRAFT][release/cd] (#6838)
```
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md:3]:
```suggestion
title: Changes to the unit in the data point template is now reflected in the info gauge widget view.
```

File: content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60261
+version: 1020.14.10
+---
+Changes to unit in the dp template is now reflected in the info gauge widget view. (#6823) [GRAFT][release/cd] (#6838)
```
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md:17]:
```suggestion
In the past, when changing the unit of a data point template, the change was not reflected in the info gauge widget which was using this data point. This inconsistent behavior has now been fixed. Now, whenever the unit is updated in a data point template, this change will also be immediately visible in the info gauge widget view.
```

File: content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60261
+version: 1020.14.10
+---
+In the past, when changing the unit of a data point template, the change was not reflected in the info gauge widget which was using this data point. This inconsistent behavior has now been fixed. Now, whenever the unit is updated in a data point template, this change will also be immediately visible in the info gauge widget view.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md:17]:
```suggestion
In the past, when changing the unit of a data point template, the change was not reflected in the "Info gauge" widgets which were using this data point. This inconsistent behavior has now been fixed. Now, whenever the unit is updated in a data point template, this change will also be immediately visible in the respective "Info gauge" widgets.
```

File: content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Changes to the unit in the data point template is now reflected in the info gauge widget view.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-14-10-changes-to-unit-in-the-dp-template-is-now-reflected-in.md:3]:
```suggestion
title: Changing the unit in a data point template is now correctly reflected in related "Info gauge" widgets
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**